src/pages/dtrader/index.tsx

```diff
@@ -72,7 +72,7 @@ const PlatformContainer = styled.div`
     padding: 8rem 0;
 `
 const Dtrader = () => {
-    const [is_mobile, setMobile] = useState('')
+    const [is_mobile, setMobile] = useState<boolean>(false)
```

Comment:
```suggestion
    const [is_mobile, setMobile] = useState(false)
```
we can remove this and let TS infer primitive types.